config/base/xnamespace/templates/sa-rbac.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: kubernetes.crossplane.io/v1alpha2
+kind: Object
+metadata:
+  annotations:
+    gotemplating.fn.crossplane.io/composition-resource-name: serviceaccount
+spec:
+  forProvider:
+    manifest:
+      apiVersion: v1
+      kind: ServiceAccount
+      metadata:
+        name: namespace-manager
+        namespace: {{ .context.namespace }}
+  providerConfigRef:
+    name: eaas-kubernetes-provider-config
+
+---
+apiVersion: kubernetes.crossplane.io/v1alpha2
+kind: Object
+metadata:
+  annotations:
+    gotemplating.fn.crossplane.io/composition-resource-name: rolebinding
+spec:
+  forProvider:
+    manifest:
+      apiVersion: rbac.authorization.k8s.io/v1
+      kind: RoleBinding
+      metadata:
+        name: namespace-manager-rb
+        namespace: {{ .context.namespace }}
+      subjects:
+        - kind: ServiceAccount
+          name: namespace-manager
+          namespace: {{ .context.namespace }}
+      roleRef:
+        kind: ClusterRole
+        name: edit
+        apiGroup: rbac.authorization.k8s.io
```

Comment:
Using this cluster role is a temporary implementation. I plan to restrict it further but in a followup PR.